File: vue/src/api/email.js
import request from '@/utils/request'

export function testEmail(data) {
  return request({
    url: '/email/test',
    method: 'post',
    data: data
  })
}

export function saveEmail(data) {
  return request({
    url: '/email/save',
    method: 'post',
    data: data
  })
}

export function getEmailStatus() {
  return request({
    url: '/email/get',
    method: 'post'
  })
}

File: go/routegroup/system/email.go
package system

import (
	"itflow/handle"
	"itflow/internal/email"

	"github.com/hyahm/xmux"
)

// Email 邮件相关路由组
var Email *xmux.RouteGroup

func init() {
	Email = xmux.NewRouteGroup()

	Email.Post("/email/test", handle.TestEmail).BindJson(&email.Email{})
	Email.Post("/email/save", handle.SaveEmail).BindJson(&email.Email{})
	Email.Post("/email/get", handle.GetEmail)

}
